zipline/algorithm.py

```diff
@@ -492,25 +489,32 @@ def _create_clock(self):
         """
         If the clock property is not set, then create one based on frequency.
         """
+        trading_o_and_c = self.trading_calendar.schedule.ix[
+            self.sim_params.sessions]
+        market_closes = trading_o_and_c['market_close'].values.astype(
+            'datetime64[ns]').astype(np.int64)
```

Comment:
Does the first `astype` do something useful, if we immediately `astype` the result?